README.md

```diff
@@ -74,45 +74,34 @@ directions to start developing/testing.
 
    Please see "default_settings.py" for an example of how a settings file should look.
 
-4. Set the following two environment variables so that Flask knows which app to run and
-   to do so in the "development" mode.
+4. Start the app.
     ```sh
-        $ export FLASK_APP=aubrey_transcription
-        $ export FLASK_ENV=development
+        $ flask --app aubrey-transcription run --debug
```

Comment:
As-is for me, this command results in `Error: Could not import 'aubrey-transcription'.` Swap dash for underscore.
```suggestion
        $ flask --app aubrey_transcription run --debug
```